quickwit-config/resources/tests/server_config/server.toml

```diff
@@ -0,0 +1,13 @@
+metastore_uri = "postgres://username:password@host:port/db"
+
+[indexer]
+data_dir_path = "/opt/quickwit/data"
+
+[searcher]
+data_dir_path = "/opt/quickwit/data"
+peer_seeds = [ "quickwit-searcher-0.local", "quickwit-searcher-1.local" ]
```

Comment:
no port?

Reply:
I guess not in this example :) We should be able to rely on default values, right?

This makes me think I need to set up the default listening ports in the server config.